assets/js/modules/search-console/components/widgets/PopularKeywordsWidget.js

```diff
@@ -21,31 +21,120 @@
  */
 import PropTypes from 'prop-types';
 
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
 /**
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
 import { CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
+import {
+	DATE_RANGE_OFFSET,
+	MODULES_SEARCH_CONSOLE,
+} from '../../datastore/constants';
+import { generateDateRangeArgs } from '../../util';
+import { numFmt } from '../../../../util';
+import Link from '../../../../components/Link';
+import MetricTileTable from '../../../../components/KeyMetrics/MetricTileTable';
+import useViewOnly from '../../../../hooks/useViewOnly';
+import { whenKeyMetricsWidgetVisible } from '../../../../components/KeyMetrics';
+import { ZeroDataMessage } from '../common';
+
+const { useSelect, useInViewSelect } = Data;
+
+function PopularKeywordsWidget( { Widget } ) {
```

Comment:
Just an aside to point out that the IB specifies `AnalyticsTopPerformingKeywordsWidget` as the widget name but it's clearly a bit out of date, not to mention the fact that it's a Search Console widget. Just mentioning for the sake of visibility in case anyone else happens upon this, this is all good.